src/Utils/Message/ProcessMessage.php

```diff
@@ -16,6 +16,7 @@ class ProcessMessage implements \JsonSerializable
     private $identifier;
     private $vendorId;
     private $imageId;
+    private $useSearchCache;
```

Comment:
Would prefer `private $useSearchCache = true;` here instead of handling default value in the `getter`

Reply:
Fixed